core/src/main/kotlin/de/fraunhofer/iem/kpiCalculator/core/strategy/KpiCalculationStrategy.kt

```diff
@@ -103,65 +96,31 @@ internal abstract class BaseKpiCalculationStrategy : KpiCalculationStrategy {
      *   missingEdgeWeights (the sum of all failed node's edge weights), hasIncompleteResults
      *   (indicates whether the original childScores contained an Incomplete result)
      */
-    private fun processChildScores(
-        childScores: Collection<Pair<KpiCalculationResult, Double>>,
-        strict: Boolean,
-    ): SeparatedKpiResults {
-
-        val successScores = mutableListOf<Pair<KpiCalculationResult.Success, Double>>()
-        val failedScores = mutableListOf<Pair<KpiCalculationResult, Double>>()
+    private fun updateEdgeWeights(edges: Collection<KpiHierarchyEdge>, strict: Boolean) {
```

Comment:
Is the code comment still valid? It says something about "sorting" but this method does not sort anything. It updates values (as the method name correctly suggests)

Reply:
updated